code/modules/mob/living/carbon/alien/special/facehugger.dm

```diff
@@ -249,6 +252,25 @@
 
 	visible_message(span_danger("[src] curls up into a ball!"))
 
+	// chest maybe because getting slammed in the chest would knock it off your face while dead
+	AddComponent(/datum/component/knockoff, knockoff_chance = 40, target_zones = list(BODY_ZONE_HEAD, BODY_ZONE_CHEST), slots_knockoffable = slot_flags)
+
+/obj/item/clothing/mask/facehugger/allow_attack_hand_drop(mob/living/carbon/human/user)
+	if(istype(user) && ishuman(loc) && stat != DEAD)
+		if(user == loc && user.get_item_by_slot(slot_flags) == src)
+			to_chat(user, span_warning("You need help taking these off!"))
```

Comment:
Well I can tell this message is just copy+pasted from prison shoes, so let's give it a unique one

Maybe something like

```suggestion
			to_chat(user, span_userdanger("[src] is latched on too tight! Get help or wait for it to let go!"))
```